src/frontend/Komponenter/Behandling/Brev/FrittståendeBrev.tsx

```diff
@@ -152,6 +153,16 @@ const FrittståendeBrev: React.FC<Props> = ({
         });
     };
 
+    const nullstillMellomlagretBrev = (): void => {
+        axiosRequest<string, null>({
+            method: 'DELETE',
+            url: `/familie-ef-sak/api/brev/mellomlager/frittstaende/${fagsakId}`,
+        }).then(() => {
+            nullstillBrev();
```

Comment:
Hvis den feiler her burde vi kanskje populere feilmelding? 👀 Samtidig er det kanskje ikke veldig viktig

Reply:
Lurte litt på det, samtidig så er det "viktigste" at vi nullstiller for brukeren slik at vedkommede kan begynne på nytt og ikke blir "stuck"